docs/user_stories/capacity_calculation_rsc.md

```diff
@@ -0,0 +1,119 @@
+```
+
+<br />
+
+<img src="./images/Import.svg" style="vertical-align: bottom"/>
+Each input file is imported with a gateway that transforms the input file in an in-memory object representing the network. The studied network results from the merge of all TSOs' networks.
+
+```java
+Network networkBe = Importers.loadNetwork(fileBe.toString());
+Network networkNl = Importers.loadNetwork(fileNl.toString());
+networkBe.merge(networkNl);
+```
+
+<br />
+
+<img src="./images/Network_merging.svg" style="vertical-align: bottom"/>
+A topological merge of the TSOs' networks is done. The following lines of code come from [powsybl-tutorials](https://github.com/powsybl/powsybl-tutorials/tree/master/cgmes) and illustrated this functionality.
+
+```java
+networkBe.merge(networkNl);
+```
+
+<br />
+
+<img src="./images/Compute_LF.svg" style="vertical-align: bottom"/>
+Then, flows are computed with a load flow simulator. In this tutorial, we use Hades2, which is a non open source software, but available in a freeware mode for experimental purposes. For more details, please visit this [page](https://rte-france.github.io/hades2/features/loadflow.html) to learn about Hades2. A `LoadFlow` object is created through a factory, here a `Hades2Factory`, which needs as input arguments a on-memory network and a computation manager `computationManager` (here defined locally by default).
+
+```java
+ComputationManager computationManager = LocalComputationManager.getDefault();
+LoadFlow loadFlow = new Hades2Factory().create(networkBe, computationManager, 0);
+```
+
+A loadflow is run on the working variant of the network with a set of parameters. The default parameters are listed [here](../configuration/parameters/LoadFlowParameters.md). Here angles are set to zero and tensions are set to one per unit. We also create a new variant to store the resulted flows. Note that a network variant is close to a state vector and gathers variables such as injections, productions, tap positions, states of buses, etc.
+
+```java
+LoadFlowParameters loadFlowParameters = new LoadFlowParameters().setVoltageInitMode(LoadFlowParameters.VoltageInitMode.DC_VALUES);
+LoadFlowResult result = loadFlow.run(VariantManagerConstants.INITIAL_VARIANT_ID, loadFlowParameters).join();
+```
+<br />
+
+<img src="./images/Compute_Sensitivity.svg" style="vertical-align: bottom"/>
+The sensitivity computation module is dedicated to compute linearized impact of network small variations on some elements state variables. The sensivity computation is fully described [here](../sensitivity/index.md). In this user story, we use this module to compute all coefficients of the cost function. The model behind is Hades2, which is a non open source software, but available in a freeware mode for experimental purposes. For more details, please visit this [page](https://rte-france.github.io/hades2/features/loadflow.html) to learn about Hades2.
```

Comment:
Instead of

> The model behind is Hades2,

I suggest:

> Since there is no fully functional open source sensitivity computation module integrated to powsybl for the time being, we use Hades2 for the purpose of the tutorial,